src/simd_128_u16.h

```diff
@@ -140,7 +139,7 @@ inline m128i mul(m128i a, m128i b, aint16 card)
     return mul(a, b);
 }
 
-/** Negatize elements of buffers
+/** Apply an element-wise negation to a buffer
```

Comment:
Since it's a single line comment, you can write `/// Apply an element-wise negation to a buffer.`